js/ad_manager_vast.js

```diff
@@ -2657,12 +2659,12 @@ OO.Ads.manager(function(_, $) {
             break;
           // case: hh:mm:ss.mmm | hh:mm:ss
           case /^\d{2}:\d{2}:\d{2}\.000$|^\d{2}:\d{2}:\d{2}$/.test(adBreak.timeOffset):
```

Comment:
Since you're in here already.... can you change this to some if/else statements? I just have a fear that some browser is going to interpret switch statements differently and we'll end up with some bugs.